community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,44 @@
+# Best practices for managing onnx issues
+These best practices apply to all onnx ecosystem repos under SIG governance.
```

Comment:
are these best practices or requirements?

"enforce" and "must" imply requirements.

if they are proposed to be requirements, then you should review with all repo owners and SIGs